test/helpers/export-csv-helper.js

```diff
@@ -40,6 +40,16 @@ module.exports.ARMS_EXPORT_CSV = {
     '"NPS North West","Cumbria","Rehabilitation","19-9-2018","CASEREF30065","A.N. Offender Manager N01B324NPSM","PO","Community","16-12-2016","16-12-2020"'
 }
 
+module.exports.T2A_EXPORT_CSV = {
```

Comment:
Could this test be updated to reflect the changes to the CSV header and the columns that have been removed? Thanks